pkg/compiler/transformers/k8s/workflow.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/lyft/flytepropeller/pkg/compiler/common"
```

Comment:
Also note, you will need to `dep ensure -update` in flyte admin to pull in this change

Reply:
yep!